interview-questions.md

```diff
@@ -7,46 +7,46 @@ Then, research the question to expand on your answer. Even if you feel you have
 
 1. In a model called Animal that has_many Sightings, what is the name of the foreign key? Would the foreign key be part of the Animal model or the Sightings model?
 
-  Your answer:
+  Your answer: The foreign key would be called animal_id, it would be a column in the Sightings model to identify the animal that sighting is identified with. 
 
-  Researched answer:
+  Researched answer: Pretty much that. 
 
 
 
 2. Which routes must always be passed params and why?
 
-  Your answer:
+  Your answer: Show, create, update, and delete. These routes are calling methods that act on a specific entry in the database. They need to know which entry to run the method on. 
 
-  Researched answer:
+  Researched answer: Or create a new one. The create method needs the parameters for the new entry. There's also routes that we haven't gone over yet like search. 
 
 
 
 3. Based on your knowledge of Rails conventions, write a route for a webpage called "game" that takes in a parameter called "guess" with a controller called "main".
 
-  Your answer:
+  Your answer: get 'main/game/:guess' => 'main#game'
 
 
 
 4. Name three rails generator commands. What is created by each?
 
-  Your answer:
+  Your answer: rails g model - generates a new model or table for our database, rails g resources - generates the RESTful routes for all basic CRUD functions, rails g migration - generates a migration file that you can edit if you need to change the shape of your database. 
```

Comment:
Yup! I would also mention the other things that generate resource creates for you.